system/argtable3/argtable3.patch

```diff
@@ -0,0 +1,512 @@
+diff -ur argtable3/argtable3.c argtable3-v3.0.0/argtable3.c
+--- argtable3/argtable3.c	2020-12-14 21:55:38.861316071 +0800
++++ argtable3-v3.0.0/argtable3.c	2020-12-14 21:52:37.189316071 +0800
+@@ -72,13 +72,13 @@
+ 
+ enum
+ {
+-    EMINCOUNT = 1,
+-    EMAXCOUNT,
+-    EBADINT,
+-    EOVERFLOW,
+-    EBADDOUBLE,
+-    EBADDATE,
+-    EREGNOMATCH
++    AEMINCOUNT = 1,
```

Comment:
Why need rename the error number?

Reply:
The defined enum structure conflicts with the macro definition in other places:
https://github.com/apache/incubator-nuttx/blob/master/include/errno.h#L303